### PR TITLE
[CI:DOCS] Create release notes for v4.4.0

### PR DESCRIPTION
**Pull request by ashley-cui**
Signed-off-by: Ashley Cui <acui@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by baude** [comment]
LGTM

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,150 @@
...
+- The `podman build` command now supports the `--group-add` option.
+- A new command, `podman network update` has been added, which updates networks for containers and pods.
+- The `podman network create` command now supports a new option, `--network-dns-server` sets the DNS servers that this network will use.
+- The `podman kube play` command now accepts the`--publish` option, in order to set or override port publishing.
+- The `podman inspect` command now returns an error field ([#13729](https://github.com/containers/podman/issues/13729)).
+- The `podman update` command now accepts the `--pids-limit` option, which adds the functionality to update the PIDs limit for a container ([#16543](https://github.com/containers/podman/issues/16543)).
+- Podman now supports container names beginning with a '/' to match Docker behaviour ([#16663](https://github.com/containers/podman/issues/16663)).
+- The `podman events` command now supports "die" as a value (mapping to "died") to the `--filter option, for better Docker compatibility ([#16857](https://github.com/containers/podman/issues/16857)).
```
Comment (Luap99) [inline RELEASE_NOTES.md:13]:
```suggestion
- The `podman events` command now supports "die" as a value (mapping to "died") to the `--filter option, for better Docker compatibility ([#16857](https://github.com/containers/podman/issues/16857)).
```
```suggestion
- The `podman events` command now supports "die" as a value (mapping to "died") to the `--filter` option, for better Docker compatibility ([#16857](https://github.com/containers/podman/issues/16857)).
```
just a small quoting issue

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,150 @@
...
+
+### Quadlet
+- Introduce Quadlet, a new systemd-generator that can be used to easily write and maintain systemd services using Podman.
+- Quadlet now features a new option, `-dryrun`, which prints out parsed unit files to stdout instead of creating the files.
+- Allow users to create podman networks using quadlet and link them to services created via .container and .kube files ([#16596](https://github.com/containers/podman/issues/16596)).
+- Quadlet now supports running `podman kube play` via .kube files.
+
+### Misc
+- CNI is being deprecated from Podman and support will be dropped at a future date. Netavark is now advised and is the default network backend for Podman.
+- The network name `pasta` is deprecated and support for it will be removed in the next major release.
```
Comment (Luap99) [inline RELEASE_NOTES.md:132]: I wonder if we should use a separate deprecated section to make it more clear to end users who do not read the full thing. cc @mheon

**Comment by ashley-cui** [comment]
Since I have to re-push anyway I'll close this PR and just tack this commit onto the release PR on the branch.

**Pull request closed without merging**